apps/web/components/team/screens/Team.tsx

```diff
@@ -37,7 +37,7 @@ const Team = ({ team }: TeamPageProps) => {
     );
 
     return (
-      <Link key={member.id} href={`/${member.username}`}>
+      <Link key={member.id} href={`/${member.username}`} passHref>
```

Comment:
Div's don't have an `href` attribute, why are we passing it here?

Reply:
Same here